daemon/daemon.go

```diff
@@ -1085,7 +1085,10 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	}
 
 	// Set up ipam conf after init() because we might be running d.conf.KVStoreIPv4Registration
-	log.Info("Initializing IPAM")
+	log.WithFields(logrus.Fields{
+		logfields.V4Prefix: dp.LocalNodeAddressing().IPv4().AllocationCIDR(),
+		logfields.V6Prefix: dp.LocalNodeAddressing().IPv6().AllocationCIDR(),
```

Comment:
Won't this panic if `enable-ipv6: false`?

Reply:
Good catch @aanm, +1.  am weary of doing a lot of `a.b.c.d` in the code because it has bitten us in the past in cases where `c` is nil.

Reply:
I was worried about this and tested this. It returns "<nil>"